pkgs/dartpad_ui/lib/console.dart

```diff
@@ -118,6 +119,61 @@ class _ConsoleWidgetState extends State<ConsoleWidget> {
                   ],
                 ),
               ),
+              // If there is a JS error, and the app type is Flutter, show a
+              // "File an issue" button. JS errors are normal when executing
+              // Dart (non-Flutter) code.
+              if (widget.output.hasError &&
+                  widget.output.hasJavascriptError &&
+                  appModel.appIsFlutter.value == true)
+                Align(
```

Comment:
Can you extract this into a separate 'file an issue' widget? To help make the code + build method more manageable.